src/Compilers/CSharp/Test/Emit3/PartialEventsAndConstructorsTests.cs

```diff
@@ -794,7 +794,7 @@ partial C() { }
             }
             """;
         CreateCompilation(source).VerifyDiagnostics(
-            // (3,13): error CS9276: Partial member 'C.C()' must have a definition part.
+            // (3,13): error CS9401: Partial member 'C.C()' must have a definition part.
             //     partial C() { }
             Diagnostic(ErrorCode.ERR_PartialMemberMissingDefinition, "C").WithArguments("C.C()").WithLocation(3, 13),
```

Comment:
>// (3,48): error CS0238: 'I.E' cannot be sealed because it is not an override [](http://example.com/codeflow?start=16&length=77)

I think this error is unexpected. `sealed` on interface members means non-virtual.

Reply:
Let's cover other accessibility modifiers (it looks like we cover `public` and `private`)

Reply:
This comment is placed on unchanged code but judging by the quoted text `// (3,48): error CS0238: 'I.E' cannot be sealed because it is not an override`, this comment was intended for `InInterface_Sealed_Private`.